django/utils/functional.py

```diff
@@ -288,15 +288,20 @@ def __init__(self):
         self._wrapped = empty
 
     def __getattribute__(self, name):
-        value = super().__getattribute__(name)
-        # If attribute is a proxy method, raise an AttributeError to call
-        # __getattr__() and use the wrapped object method.
-        if not getattr(value, "_mask_wrapped", True):
-            raise AttributeError
+        __getattribute__ = super().__getattribute__
+        found = True
+        try:
+            value = super().__getattribute__(name)
```

Comment:
I _presume_ this could be `value = __getattribute__(name)` given the binding prior ... but this could also be one of those occasions where things are dynamic enough that super gets confused so it's necessary to do it this way?

Reply:
`value = __getattribute__(name)` should work fine. I forgot to change it after adding the `__getattribute__` variable.

Reply:
Good point! This seems like an oversight to me. Tests pass locally when using `__getattribute__`